src/button/component.jsx

```diff
@@ -187,6 +192,14 @@ export let Button : Component<ButtonOptions> = create({
             queryParam: true
         },
 
+        isFundingThrottleEnabled: {
+            type:     'boolean',
```

Comment:
Can we add `queryParam: true` -- since the server-side will need to know to render this experience too.